### PR TITLE
[Enhancement](group commit) Add bvar and log for group commit

### PR DESCRIPTION
**Pull request by Yukang-Lian**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: be/src/vec/sink/group_commit_block_sink.cpp [hunk 1]
```diff
@@ -141,12 +144,9 @@ Status GroupCommitBlockSink::send(RuntimeState* state, vectorized::Block* input_
         return status;
     }
     SCOPED_TIMER(_profile->total_time_counter());
-    // update incrementally so that FE can get the progress.
-    // the real 'num_rows_load_total' will be set when sink being closed.
-    state->update_num_rows_load_total(rows);
```
Comment (mymeiyi) [inline be/src/vec/sink/group_commit_block_sink.cpp:146]: the stream load row count is error if remove this line?

**Comment by Yukang-Lian** [comment]
run buildall

**Review by mymeiyi — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**